lib/provisioning.js

```diff
@@ -0,0 +1,17 @@
+const provisioningWatermarks = {
+  'enterprise': ['<key>ProvisionsAllDevices</key>'],
```

Comment:
@trufae is it always the case that the `<key>Prov...` has no spaces? what if there are spaces, shouldn't it still identity?

Reply:
Yes, this file is provided by Apple, and it's like this in all the cases i have tested. But maybe they can switch to binary plist and this will not work, so the good fix will require to parse the blob extract the plist and parse it.

Reply:
The embedded mobileprovision is constructed in ASN1/DER form, that plist is embedded inside an OCTET STRING primitive of the pkcs7-data data object of it. we can probably . use the asn1js nodejs module here. and avoid problems if the user decides to provide a custom mobileprovision with those strings to fool the provisioning type detection code.